### PR TITLE
update observers description per issue #234

### PR DESCRIPTION
**Pull request by jorydotcom**
incorporates language from Mike Dolan and Michael Dawson

### CODE REVIEW

**Comment by jorydotcom** [comment]
Per CPC meeting on 7/10, moved the language update to the README so we don't have to send a charter change in for board approval.

File: README.md [hunk 1]
```diff
@@ -93,7 +93,17 @@ According to the [CPC Charter](https://github.com/openjs-foundation/cross-projec
 - Myles Borins (@MylesBorins)
 - Sendil Kumar (@sendilkumarn)
 
-<!-- ### Observers -->
+### Observers
+
+Anyone who participates in an OpenJS Foundation project can be an Observer.
```
Comment (MylesBorins) [inline README.md]: I was under the impression that we wanted to go even more liberal with this...

```suggestion
Anyone can be an Observer.
```

Comment (mhdawson) [reply README.md]: +1

Comment (joesepi) [reply README.md]: +1

**Pull request merged**